observer/observer.go

```diff
@@ -0,0 +1,96 @@
+package observer
+
+import (
+	"strconv"
+
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+
+	_ "github.com/letsencrypt/boulder/observer/probers/dns"
+	_ "github.com/letsencrypt/boulder/observer/probers/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	statMonitors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_monitors",
+			Help: "details of each configured monitor",
+		},
+		[]string{"name", "kind", "valid"},
```

Comment:
Given that we can't set the `name` field for invalid monitors, I'd drop the name field entirely. (See also my comment that it appears we're not incrementing this metric at all for valid monitors, so it isn't currently capturing an accurate count of all monitors anyway.)